core/consensus/babe/src/lib.rs

```diff
@@ -847,10 +851,11 @@ fn claim_slot(
 }
 
 #[cfg(test)]
-#[allow(dead_code, unused_imports)]
+#[allow(dead_code, unused_imports, deprecated)] //FIXME
```

Comment:
what's deprecated here? does it need an issue?

Reply:
same reason. marked it with the issue